apps/nextra/pages/en/build/indexer/indexer-sdk/documentation.mdx

```diff
@@ -0,0 +1,5 @@
+---
+title: "Documentation"
+---
+
+# Documentation
```

Comment:
If there's not going to be anything on this page, you can remove the file.  Similar to the installation section in the CLI